content/en/docs/releases/kubeflow-1.6.md

```diff
@@ -0,0 +1,186 @@
+      </tr>
+      <tr>
+        <td>Central Dashboard</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.6.0/components/centraldashboard">v1.6.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Jupyter Web App</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.6.0/components/crud-web-apps/jupyter">v1.6.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Kubeflow Access Management API</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.6.0/components/access-management">v1.6.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Notebook Controller</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.6.0/components/notebook-controller">v1.6.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Profile Controller</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.6.0/components/profile-controller">v1.6.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Tensorboard Controller</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.6.0/components/notebook-controller">v1.6.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Tensorboard Web App</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.6.0/components/crud-web-apps/volumes">v1.6.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Volumes Web App</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.6.0/components/crud-web-apps/tensorboards">v1.6.0</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Pipelines Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="2" class="align-middle">Pipelines Working Group</td>
+        <td>Kubeflow Pipelines</td>
+        <td>
+          <a href="https://github.com/kubeflow/pipelines/releases/tag/2.0.0-alpha.3">v2.0.0-alpha.3</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Kubeflow Pipelines Tekton</td>
+        <td>
+          <a href="https://github.com/kubeflow/kfp-tekton/releases/tag/v1.2.1">v1.2.1</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Serving Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="1" class="align-middle">Serving Working Group</td>
+        <td>KServe</td>
+        <td>
+          <a href="https://github.com/kserve/kserve/releases/tag/v0.8.0">v0.8.0</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Training Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="1" class="align-middle">Training Working Group</td>
+        <td>Training Operator</td>
+        <td>
+          <a href="https://github.com/kubeflow/training-operator/releases/tag/v1.5.0">v1.5.0</a>
+        </td>
+      </tr>
+  </tbody>
+</table>
+</div>
```

Comment:
Non-blocking: It will be great to have a list of K8S versions we support in this release.

Reply:
that's a great idea! I updated the page to include all dependencies that we tracked for the release. PTAL https://deploy-preview-3332--competent-brattain-de2d6d.netlify.app/docs/releases/kubeflow-1.6/

Reply:
Looks good! Thank you Anna!